spec/controllers/families_controller_spec.rb

```diff
@@ -25,4 +25,34 @@
       end
     end
   end
+
+  describe "DELETE /families/:id" do
+    context "When the family id exsist" do
+      it "destroys that family" do
+        family = create(:family)
+
+        expect { delete :destroy, params: { id: family.id } }.to change(Family, :count).by(-1)
+      end
+
+      it "flashes a notice message and render the family list" do
```

Comment:
A suggestion to change the wording to be:
`it "sets a flash notice message and redirects to index"`